### DISCUSSION TITLE
Mortal Kombat 11

### DISCUSSION THREAD

**Discussion by MK11Shadow** [skyline-emu/skyline#580] [Game Requests]
Mortal kombat 11. I love that game. It would be awesome if skyline could handle it.

**Comment by yhak22** [comment]
i'm with you man it would be great to play it on the go 
unlike MK for android it's great but it's not satisfying  me

**Reply by MK11Shadow** [reply]
Exactly. We want to do some satisfying combos to kick some ***..
